package.json

```diff
@@ -13,6 +13,7 @@
   "dependencies": {
     "@openzeppelin/contracts": "5.3.0",
     "@prb/math": "4.1.0",
+    "@sablier/devkit": "github:sablier-labs/devkit#main",
```

Comment:
Why isn't it a part of `devDependencies`?

Reply:
Huh. Mistake!